index.html

```diff
@@ -1,400 +1,375 @@
+                  fill="#5E6C84" />
+              </svg>
+            </li>
+          </ul>
+          <ul class="lang-list">
+            <li>
+              <img src="img/js.png" alt="JavaScript">
+              JavaScript
+            </li>
+            <li>
+              <img src="img/html.png" alt="HTML">
+              HTML
+            </li>
+            <li>
+              <img src="img/css3.png" alt="CSS3">
+              CSS
+            </li>
+          </ul>
+          
+          <ul class="tools-title b-bottom">
+            <li class="title">Frameworks</li>
+            <li>
+              <svg width="8" height="15" viewBox="0 0 8 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" clip-rule="evenodd"
+                  d="M0.292893 0.439622C0.683417 0.0490974 1.31658 0.0490974 1.70711 0.439622L7.70711 6.43962C8.09763 6.83015 8.09763 7.46331 7.70711 7.85384L1.70711 13.8538C1.31658 14.2444 0.683417 14.2444 0.292893 13.8538C-0.0976311 13.4633 -0.0976311 12.8301 0.292893 12.4396L5.58579 7.14673L0.292893 1.85384C-0.0976311 1.46331 -0.0976311 0.830146 0.292893 0.439622Z"
+                  fill="#505F79" />
+              </svg>
+            </li>
+          </ul>
           
+          <ul class="tools-title b-bottom">
+            <li class="title">Skills</li>
+            <li>
+              <svg width="8" height="15" viewBox="0 0 8 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" clip-rule="evenodd"
+                  d="M0.292893 0.439622C0.683417 0.0490974 1.31658 0.0490974 1.70711 0.439622L7.70711 6.43962C8.09763 6.83015 8.09763 7.46331 7.70711 7.85384L1.70711 13.8538C1.31658 14.2444 0.683417 14.2444 0.292893 13.8538C-0.0976311 13.4633 -0.0976311 12.8301 0.292893 12.4396L5.58579 7.14673L0.292893 1.85384C-0.0976311 1.46331 -0.0976311 0.830146 0.292893 0.439622Z"
+                  fill="#505F79" />
+              </svg>
+            </li>
+          </ul>
         </div>
-      </section>
-    </main>
-  </body>
+      </div>
+    </section>
+
+    <section class="contact-wrapper">
+      <div class="contact-container">
+        <h1 class="intro-text">Contact me</h1>
+        <p>
+          If you have an application you are interested in developing,
+          a feature that you need built or a project that needs
+          coding. I’d love to help with it
+        </p>
+      <form action="https://formspree.io/f/mgedddej" method="post">
+        <input name="Name" id="text" type="text" placeholder="Your name" maxlength="30" required>
+        <input name="Email" id="email" type="email" placeholder="Your email address" required>
+        <textarea name="comment" placeholder="Write your message here" maxlength="500" required></textarea>
+        <button type="submit" value="Submit" class="btn">Get In Touch</button>
+      </form>
```

Comment:
Your design is really good 👏👏. I noticed that your button is not aligned to the center on the Desktop if you notice that is the case. Please make this change to keep in line with the Figma 👍.

Figma:
<img width="291" alt="image" src="https://user-images.githubusercontent.com/7243132/171687962-b3c618a0-8296-4a0a-bcc6-87666857cfcc.png">

Your design:
<img width="564" alt="image" src="https://user-images.githubusercontent.com/7243132/171688042-49615f05-1e66-4560-afb6-d13424acf84a.png">